### PR TITLE
Fixes for Prow configs

### PR DESCRIPTION
**Pull request by adrcunha**
* unset kubectl context after updating Prow cluster to avoid further interactions with the cluster by mistake
* move LimitRange to cluster.yaml

### CODE REVIEW

File: ci/prow/Makefile [hunk 1]
```diff
@@ -19,27 +19,36 @@ JOB_NAMESPACE ?= test-pods
 
 PROW_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 TESTGRID_DIR := $(PROW_DIR)/../testgrid
+UNSET_CONTEXT := kubectl config unset current-context
 
 get-cluster-credentials:
 	gcloud container clusters get-credentials "$(CLUSTER)" --project="$(PROJECT)" --zone="$(ZONE)"
 
+unset-cluster-credentials:
```
Comment (chaodaiG) [inline ci/prow/Makefile]: nit, would be nice to reuse this target instead of `UNSET_CONTEXT`

**Comment by chaodaiG** [comment]
/lgtm
/hold
in case you are interested in the nit I left

File: ci/prow/Makefile (hunk 1 above)
Comment (adrcunha) [reply ci/prow/Makefile]: I find `make unset-cluster-credentials` output too verbose and hard to read, thus the trick.

**Pull request merged**